### DISCUSSION TITLE
create tables?

### DISCUSSION THREAD

**Discussion by tleclaire** [ShapeCrawler/ShapeCrawler#197] [Q&A] [answered]
Can I create tables in slides with this?

Best regards
Thomas

**Comment by AdamShakhabov** [comment]
Hello Tomas.
No, this feature is not implemented yet.

**Comment by AdamShakhabov** [comment, marked as answer]
The new `IShapeCollection.AddTable()` method has been added to add table on a slide. The feature will be available in the next v0.39.0 version.
